CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - preserve native types when doing jinja substitution
 - inventory argument to the CLI is optional
 - select works with null
+- a race condition between threads in drools rule engine
```

Comment:
```suggestion
- fixed a race condition between threads in drools rule engine
```

Reply:
@Alex-Izquierdo The Section title already has a Fixed and is listing 1 fix at time so I didn't want to repeat fixed.
https://github.com/ansible/ansible-rulebook/blob/main/CHANGELOG.md#fixed

Reply:
ah, ok, I didn't see it in the preview mode. sorry